gomp-git-PKGBUILD/PKGBUILD

```diff
@@ -0,0 +1,64 @@
+#
+# Maintainer: Mahmoud Mohamed (00xWolf) <mmsaeed509@gmail.com> , <https://github.com/mmsaeed509>
+#
+
+pkgname=gomp-git
+pkgver=r329.0c0cbbf
+pkgrel=3
+pkgdesc='MPD client inspired by ncmpcpp written in GO.'
+arch=('any')
+url="https://github.com/aditya-K2/gomp"
+license=('GPL3')
+
+makedepends=(
+  'git'
+   'go'
+   )
+
+source=(
+  "${pkgname}::git+${url}"
+  "config.yml"
+  "cover.jpg"
+  )
+
+sha256sums=(
+  'SKIP'
+  'SKIP'
+  'SKIP'
+  )
+
+groups=('Exodia-TUI-Apps')
```

Comment:
I don't think I want to add `gomp` to this group. Everything else looks good.

Reply:
this group is for `Exodia OS` as it supports a lot of TUI Apps